index.html

```diff
@@ -44,30 +44,30 @@ <h1>I'm Anicet<br />Glad to see <br />you!</h1>
                 <div class="connect"><a href="">LET'S CONNECT</a></div>
```

Comment:
- [ ] Please, consider giving your page title a name like `Mark's portfolio` or `Mark full-stack web developer`. In your case it is "Mobile version" - it should briefly describe the content of the page.